### DISCUSSION TITLE
twitter_username and github_username in _config.yml does not go into social media links

### DISCUSSION THREAD

**Discussion by bodote** [mmistakes/minimal-mistakes#2691] [Q&A] [answered]
I have twitter_username and github_username set in _config.yml. 
```
...
twitter_username: BodoTe
github_username: bodote
...
```

I expect that the social media links get these user names appended automaticaly , which seems not to be the case.

do i need to add the usernames manually at _config.yml here below at `url:`or should this work automaticaly from the `twitter_usernam`given above ?:

```
...
author:
  name   : "Bodo T."
    avatar : "/assets/images/bio-photo.jpg"
    bio    : "Developing software since 1980"
    links:
      - label: "Twitter"
        icon: "fab fa-fw fa-twitter-square"
        url: "https://twitter.com/"
       #or url: "https://twitter.com/BodoTe" ?
      - label: "GitHub"
        icon: "fab fa-fw fa-github"
        url:"https://github.com/"
        # or url:"https://github.com/bodote" ?
...
```

if not: for what reason do  set the `twitter_username` in _config.yml ?

**Comment by gamue** [comment]
Where did you find documentation about the properties `twitter_username` and `github_username`? I can't find them anywhere and for this theme adjusting the `links`-section is the way to go.

**Comment by mmistakes** [comment, marked as answer]
`twitter_username` and `github_username` are not site variables used by this theme.

I believe you're picking them up from older version of Jekyll's default theme, [**Minima**](https://github.com/jekyll/minima). Themes are generally interchangeable unless they've adopted the same naming conventions.

**Comment by bodote** [comment]
Well, they just are in my _config.yml , but I am not sure from what source these entrys really are...
Seems I need to dig deeper into jekyll in general first, before using other themes.
Nevertheless, thank you for your reply!
